### DISCUSSION TITLE
v5.2.5

### DISCUSSION THREAD

**Discussion by madeye** [shadowsocks/shadowsocks-android#2758] [General]
Bug fixes for the local DNS resolver. (#2751 #2754 #2757)

<hr /><em>This discussion was created from the release <a href='https://github.com/shadowsocks/shadowsocks-android/releases/tag/v5.2.5'>v5.2.5</a>.</em>

**Comment by mmofaha** [comment]
I would like to request for button/menu of :

1. REORDER BY DELAY by test result - for easily choose the best connection
2. DELETE ALL / REMOVE ALL configuration server list for easy to load a new one
or you can see the triple dot menu list from SagerNet and AnXray for reference

TQ

**Comment by mmofaha** [comment]
I like it Shadowsocks. It run smoothly.
Can u please add a menu of "REORDER BY DELAY" so easily to select which proxy is best conncection at that time.
SagerNet and AnXray have this menu inside it.
TQVM
